src/MAUI/Maui.Samples/Samples/Layers/DefineClusteringFeatureReduction/DefineClusteringFeatureReduction.xaml

```diff
@@ -0,0 +1,87 @@
+﻿<ContentPage x:Class="ArcGIS.Samples.DefineClusteringFeatureReduction.DefineClusteringFeatureReduction"
+             xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:esriTK="clr-namespace:Esri.ArcGISRuntime.Toolkit.Maui;assembly=Esri.ArcGISRuntime.Toolkit.Maui"
+             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui">
+    <Grid ColumnDefinitions="*, Auto" RowDefinitions="Auto, *">
```

Comment:
Can you use the Dynamic Grid resource for this? That will fix the landscape orientation issue once #1317 is completed.